tests/extract-css.js

```diff
@@ -125,22 +126,30 @@ describe("Test CSS properties extraction", function() {
   this.slow(5000);
   this.timeout(10000);
   let browser;
+  let extractCSSCode;
   before(async () => {
+    // Convert the JS module to a JS script that can be loaded in Puppeteer
+    // without having to provide a URL for it (tests run in "about:blank" pages)
+    const bundle = await rollup.rollup({
```

Comment:
this may be obvious, but I don't understand why we need rollup here?

Reply:
OK, having tried various approaches, I now understand that there isn't a simple way to plug our module in a reusable context in puppeteer - this sounds like a gap in the platform (in particular, there is no way to import functions from an in-line `<script type=module></script>`), but that's for a different yak shaving day.